src/components/Header.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Header from './Header'
+
+export default {
+  title: 'Components/Header',
+  component: Header,
+}
+
+export const Text = () => <Header>Header Text</Header>
+
+//const Template = (args) => <Header {...args} />
```

Comment:
Please delete comments as we always want the main branch to be clear of comments.